src/main/java/cms/gov/madie/measure/config/MigrateSupplementalDataElementsChangeUnit.java

```diff
@@ -0,0 +1,74 @@
+package cms.gov.madie.measure.config;
+
+import java.util.List;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
+
+import cms.gov.madie.measure.repositories.MeasureRepository;
+import gov.cms.madie.models.measure.Measure;
+import gov.cms.madie.models.measure.MeasureMetaData;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@ChangeUnit(id = "migrate_supplemental_data_elements", order = "1", author = "madie_dev")
+public class MigrateSupplementalDataElementsChangeUnit {
+
+  @Setter private List<Measure> tempMeasures;
+
+  @Execution
+  public void migrateSupplementalDataElements(MeasureRepository measureRepository)
+      throws Exception {
+    log.debug("Entering migrateSupplementalDataElements()");
+
+    List<Measure> measureList = measureRepository.findAll();
+    if (CollectionUtils.isNotEmpty(measureList)) {
+      setTempMeasures(measureList);
+      measureList.stream()
+          .forEach(
+              measure -> {
+                if (CollectionUtils.isNotEmpty(measure.getSupplementalData())) {
+                  StringBuffer sb = new StringBuffer();
```

Comment:
Since `forEach` will run single-threaded, recommend using StringBuilder instead of StringBuffer.
```suggestion
                  StringBuilder sb = new StringBuilder();
```

Reply:
@jkotanchik-SB that makes sense, thanks so much Joe!